bottles/backend/wine/winecommand.py

```diff
@@ -601,6 +603,21 @@ def get_cmd(
 
         if pre_script is not None:
             command = f"sh '{pre_script}' ; {command}"
+        
+        if midi_soundfont is not None:
+            command = f"""
+                # start fluidsynth in dedicated terminal
+                # (as, for some reason, fluidsynth doesn't like being run with &)
+                fluidsynth="fluidsynth --server --audio-driver=pipewire '{midi_soundfont}'"
+                flatpak-spawn --host gnome-terminal -- bash -c "$fluidsynth"
+
+                # save PID to individually kill process afterwards
+                fluidsynth_pid=$(flatpak-spawn --host pidof fluidsynth | cut -d " " -f 1)
+
+                {command}
+
+                flatpak-spawn --host kill $fluidsynth_pid
```

Comment:
The whole spawn of `flatpak-spawn`s (sorry about the horrible pun) is needed for most of the commands to work on my system. It's probably not the intended behavior as the issue also happens when trying to put anything in my pre or post-scripts (lest I want to get a bunch of "command not found" errors). No idea if it's some wrong config on my part.